### PR TITLE
update caption

### LINKED ISSUE #1: Add collaborators to this repo
## Comment below to be added as a collaborator

We will all be using this repository today. Even though it's public right now, you will not be able to make changes until you're given the correct permissions. We've automated this process via the GitHub API. Once you comment, we will add you as a collaborator.

You'll start to receive a lot of emails. (❗) You can slow these down immediately by clicking the **Unwatch** button at the top of the page and selecting either the **Participating and @mentions** or **Ignore** option.

### PR DESCRIPTION
**Pull request by fredgohsman**
**updated comment in slide**
🥇 
closes #1 

- [x] task 1
- [ ] tasks 2

@sn493s

### CODE REVIEW

**Comment by vs252n** [comment]
Looks good

File: _slides/02-fredgohsman.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello, this is Fred Gohsman's slide.
```
Comment (fredgohsman) [inline _slides/02-fredgohsman.md:6]:
```suggestion
Hello, my name is Fred!
```

File: _slides/02-fredgohsman.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ layout: slide
...
```
Comment (fredgohsman) [inline _slides/02-fredgohsman.md:6]: This is nice.

File: _slides/02-fredgohsman.md [hunk 3]
```diff
@@ -3,7 +3,8 @@ layout: slide
...
+This is my Slide!
 
 ![dojocat](https://octodex.github.com/images/dojocat.jpg)
```
Comment (fredgohsman) [inline _slides/02-fredgohsman.md:9]: This needs some work.

**Review by fredgohsman — COMMENTED** [review]
Do some changes

File: _slides/02-fredgohsman.md [hunk 4]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello, this is Fred Gohsman's slide.
+This is my Slide!
```
Comment (fredgohsman) [inline _slides/02-fredgohsman.md:7]:
```suggestion
This is my new slide.
```

**Review by cangao01 — APPROVED** [review]
Approved.

**Pull request merged**